contracts/orchestrator-contracts/package.json

```diff
@@ -43,7 +43,7 @@
     "ts-node": "^10.9.1",
     "tsc-alias": "^1.8.8",
     "tsup": "^8.0.0",
-    "typescript": "^5.3.2",
+    "typescript": "5.0.4",
```

Comment:
I'm not sure we should downgrade TS. Instead I think we should ditch `ts-node` which is internally used by Serverless and is no longer maintained. I'll try and push some updates to this PR